CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ gem "capistrano", :github => "capistrano/capistrano"
 [master]: https://github.com/capistrano/capistrano/compare/v3.10.1...HEAD
 
 * Your contribution here!
+### New features:
+
+* [#1058](https://github.com/capistrano/capistrano/pull/1058): Update git plugin rake tasks - configurable max concurrent connections and wait interval settings
```

Comment:
The PR ID is #1957. The issue ID is #1058. I think we generally link to the PR ID.

Reply:
Can you please change this @grzegorzblaszczyk Will is right

Reply:
@leehambley I think I put the correct link now.